### DISCUSSION TITLE
Any way to pause/skip specific celery tasks?

### DISCUSSION THREAD

**Discussion by burner1024** [WeblateOrg/weblate#7658] [Q&A] [answered]
I'm working with API, updating units (`PATCH /api/units/${id}/`) in bulk. Each update takes about 30 seconds, and there are thousands to process. 
I noticed that each API call triggers an `weblate.trans.tasks.update_component_stats` task. I wonder if there's a way to turn it off for a time, and then run it once manually, after the update process is finished?

If there's no proper way, even something hacky like commenting out a few code lines would do. It's a one time process in my case, not something that needs to be done repeatedly.

Or any other way to speed up patching, I guess.

**Comment by nijel** [comment, marked as answer]
There is no configuration for that, you can temporarily add `return` to the top of `update_component_stats` in `weblate/trans/tasks.py`.

**Reply by burner1024** [reply]
Cool, thank you!
